cpp/src/Ice/SysLoggerI.cpp

```diff
@@ -12,7 +12,7 @@ using namespace std;
 using namespace Ice;
 using namespace IceInternal;
 
-Ice::SysLoggerI::SysLoggerI(const string& prefix, const string& facilityString) : _facility(0), _prefix(prefix)
+Ice::SysLoggerI::SysLoggerI(string prefix, const string& facilityString) : _facility(0), _prefix(std::move(prefix))
```

Comment:
Why do we keep facilityString as a reference?

Reply:
Fixed. See earlier comment.